File: examples/workflow-events/src/main/java/io/littlehorse/examples/MyWorker.java
package io.littlehorse.examples;

import io.littlehorse.sdk.worker.LHTaskMethod;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class MyWorker {

    private static final Logger log = LoggerFactory.getLogger(MyWorker.class);

    @LHTaskMethod("increment")
    public Double increment(Double currentValue) {
        log.debug("Executing task increment");
        log.info(String.valueOf(currentValue));
        return ++currentValue;
    }

}

File: sdk-dotnet/Examples/ChildThreadsForeachExample/MyWorker.cs
using LittleHorse.Sdk.Worker;

namespace ChildThreadsForeachExample;

public class MyWorker
{
    [LHTaskMethod("task-executor")]
    public async Task<string> TaskExecutor(string taskInput)
    {
        Console.WriteLine($"ok, executing task with {taskInput}");

        return $"Executed task with input: {taskInput}";
    }
}